public/locales/en-US/send.ftl

```diff
@@ -41,6 +41,8 @@ deleteFileButton = Delete file
 sendAnotherFileLink = Send another file
 // Alternative text used on the download link/button (indicates an action).
 downloadAltText = Download
+downloadsFileList = Downloads
+timeFileList = Time
```

Comment:
Sorry, I lost track of this PR. This should have a comment, "time" it's a very broad word in English.

Reply:
This `Time` refers to this header here 
![screenshot from 2018-01-23 12-23-16](https://user-images.githubusercontent.com/25258877/35261960-82da6098-0038-11e8-8b28-2269e8543edb.png)


### This initially was set as `Expires In` should I change it back.

Reply:
That's really not my call ;-)

In the meantime I opened a PR to add a localization comment.